makefile

```diff
@@ -17,6 +17,25 @@ LIB_SRC := \
 	$(shell find stdx-allocator/source -name "*.d")
 PROJECT_SRC := $(shell find src/ -name "*.d")
 SRC := $(LIB_SRC) $(PROJECT_SRC)
+
+UT_OBJ_DIR = unittest-obj
```

Comment:
doesn't this clutter the project directory / add files that may accidentally be committed to git? (not gitignored)

I think it would be better if it was in obj/unittest or if the unittest-obj folder was added to gitignore.

Reply:
You're right. I've moved it inside the `obj` directory